content/docs/getting-started/feedback.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Send feedback to Scratch Addons developers
+description: How to send feedback to Scratch Addons.
+weight: 3
+no_comments: true
+toc_title: Send Feedback
+---
+
+Have something to say about Scratch Addons, the browser extension? Let us know through the feedback form located on https://scratchaddons.com/feedback/.
+
+{{< admonition warning >}}
+
+We can't help with issues related to Scratch moderation. Instead, use the Report button on the Scratch website, or reach Scratch through their official channels.
+
+{{</ admonition >}}
+
+## Alternatives
+
+In case you need an alternative, here are some choices.
+
+- Create a issue on [the repository](https://github.com/ScratchAddons/ScratchAddons/issues).
+- Create a post on [our Discussions tab](https://github.com/ScratchAddons/ScratchAddons/discussions)
+- Send a message on [our Discord server](https://discord.gg/R5NBqwMjNc)
+- Contact one of the contributors on Discord, Twitter, or other methods
```

Comment:
By the way, maybe you could also add scratchaddons.com/feedback to this list, in case someone forgets to read the first paragraph.